src/components/target/MatchFoundComponent/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { View, Text, TouchableOpacity, Modal, ImageBackground } from 'react-native';
+import { connect } from 'react-redux';
+import { func, bool, object } from 'prop-types';
+
+import { matchFoundHeader, profileIcon } from '../../../image';
+import styles from './styles.js';
+
+export default class MatchFoundComponent extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const { skipMatch, foundMatch: { username }} = this.props;
+
+    return (
+      <Modal
+        animationType="slide"
+        transparent={true}
+        visible={true} >
+        <View style={styles.outerModal}>
+          <View style={styles.modal}>
+            <ImageBackground
+              source={matchFoundHeader}
+              style={styles.headerBackground}
+              imageStyle={styles.headerImage}
+            />
+            <Text style={styles.yayText}>Yay!</Text>
+            <Text style={styles.newMatchText}>You have a new match!</Text>
+            <View style={styles.matchProfileView}>
```

Comment:
Is this view necessary? Can't you do this:
```suggestion
            <ImageBackground>
              <Text>
            <ImageBackground/>
```

Reply:
to horizontally align image and text, not sure what would be the result of your sample code